google/cloud/forseti/services/inventory/inventory.py

```diff
@@ -303,3 +309,49 @@ def delete(self, inventory_id):
         with self.config.scoped_session() as session:
             result = DataAccess.delete(session, inventory_id)
             return result
+
+    def purge(self, retention_days):
+        """Purge the gcp_inventory data that's older than the retention days.
+
+        Args:
+            retention_days (int): Days of inventory tables to retain.
+
+        Returns:
+            str: Purge result.
+        """
+        LOGGER.debug('retention_days is: %s', retention_days)
```

Comment:
Is there any reason why these logs are DEBUG instead of INFO? Usually clients don't turn no the  debug flag and might be hard to debug if they found some of the data missing.